tests/acceptance/features/apiGraph/addUserToGroup.feature

```diff
@@ -149,3 +149,21 @@ Feature: add users to group
   Scenario: admin tries to add user to a group without sending the group
     When the administrator tries to add user "Alice" to group "" using the Graph API
     Then the HTTP status code should be "404"
+
+
+  Scenario: add multiple users to a group in a single request
+    Given the administrator has given "Alice" the role "Admin" using the settings api
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Brian    |
+      | Carol    |
+    And user "Alice" creates a group "grp1" using the Graph API
+    When the administrator "Alice" adds the following users to a group "grp1" in a single request using the Graph API
```

Comment:
```suggestion
    When the administrator "Alice" adds the following users to a group "grp1" using the Graph API
```

Reply:
Don't you think it will be nice if we differentiate single and multiple requests

Reply:
`at once` would be nice for readability.